pkgs/development/python-modules/mo_installer/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib,
+  fetchPypi,
+  buildPythonPackage,
+  setuptools-scm,
+  pyfakefs
+}:
+
+buildPythonPackage rec {
+  pname = "mo_installer";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-4EmLYtq0LryD3nWG1cHhP6EpWfqm44LmhfpQ1PKCwrY=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  # contains no tests
+  doCheck = false;
```

Comment:
looks like they have tags https://github.com/s-ball/mo_installer/tags and test in their source, do you mind trying to build from the repo?

Also, the repo looks to have 0 activity in 3  years.

Reply:
@jonringer I resolved this by removing the mo_installer package entirely. The tests for that package seem to fail and mo_installer was only a build time dependency in schematics that does not seem to be used.

Reply:
Also I was not really happy either adding a package that was not maintained like that. Thanks for the review!